src/App/ColorPalette.svelte

```diff
@@ -165,30 +137,23 @@
   }
 </style>
 
-<svelte:window on:keydown={onKeydown} on:click={clickOutside} />
-
-{#if show}
-  <!-- svelte-ignore a11y-click-events-have-key-events -->
-  <div
-    bind:this={thisComponent}
-    class="container"
-    on:click={() => (checkers = !checkers)}>
-    <div class:checkers>
-      {#each colorGroups as colorGroup}
-        <div>
-          {#each colors.filter(color => {
-            return color.match(`--color-${colorGroup}`);
-          }) as color}
-            <div style="display: inline-flex;">
-              <div
-                class:unused={!usedColors.includes(color)}
-                title={color}
-                class="color"
-                style={`background-color: var(${color});`} />
-            </div>
-          {/each}
-        </div>
-      {/each}
-    </div>
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<div class="container" on:click={() => (checkers = !checkers)}>
+  <div class:checkers>
+    {#each colorGroups as colorGroup}
+      <div>
+        {#each colors.filter(color => {
+          return color.match(`--color-${colorGroup}`);
+        }) as color}
+          <div style="display: inline-flex;">
```

Comment:
Could we maybe use svelte style directives here too?
```suggestion
          <div style:display="inline-flex">
```